run.py

```diff
@@ -106,6 +126,19 @@ def config_validator():
         app.config['TESTING'] = True
         print("Starting server in ~*TEST MODE*~")
         app.run(host='localhost')
+    elif args.backup:
+        try:
+            command = ['ls', '-l', 'issues.db']
+            if execute_command(command)[1] == 0:
+                execute_command(['mkdir', '-p', app.config['BACKUP_DEFAULT_DEST']])
+                current_ts = time.time()
+                time_stamp = datetime.datetime.fromtimestamp(current_ts).strftime('%Y-%m-%d-%H:%M:%S')
+                issue_backup_db = 'issues_backup_' + time_stamp + '.db'
```

Comment:
As is, this will keep all backups.

Maybe we could keep the previous 2 or something? (So we have 3 total)?